.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[Bug] - <Descriptive title>"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Reproduce**
+The more information you can provide us, the quicker we respond and fix the problem.
```

Comment:
I would move this line to the Additional context header.
That way this will only contain the steps to reproduce